packages/math-input/src/components/iconography/exp-2.js

```diff
@@ -26,4 +26,4 @@ class Exp2 extends React.Component {
     }
 }
 
-module.exports = Exp2;
+export default Exp2;
```

Comment:
Thankfully rollup boils away the overhead of having these icons live in separate files.